CHANGELOG.md

```diff
@@ -6,8 +6,8 @@
 
 ### Fixed
 * <How do the end-user experience this issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
- 
+* Fix a data race when preparing to deliver change notifications ([PR #5892](https://github.com/realm/realm-core/pull/5892) since 12.7.0).
```

Comment:
Could this have affected end-user applications (and how would they see that)?

Reply:
Probably not, and no idea.